.github/workflows/android.yml

```
@@ -25,6 +25,14 @@ jobs:
             all_files_access: 'OFF'
           - triplet: 'x86-android'
             all_files_access: 'OFF'
+          - triplet: 'arm64-android'
+            all_files_access: 'ON'
+          - triplet: 'arm-android'
+            all_files_access: 'ON'
+          - triplet: 'x64-android'
+            all_files_access: 'ON'
+          - triplet: 'x86-android'
+            all_files_access: 'ON'
 
```

you can simplify a bit the matrix:
```suggestion
    strategy:
      matrix:
        triplet: ['arm64-android', 'arm-android', 'x64-android', 'x86-android']
        all_files_access: ['ON', 'OFF']
```